cloudflare/resource_cloudflare_account_member_test.go

```diff
@@ -0,0 +1,40 @@
+package cloudflare
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareAccountMemberBasic(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := "cloudflare_account_mamber." + rnd
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheckAccount(t)
+			testAccPreCheckEmail(t)
+			testAccPreCheckApiKey(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testCloudflareAccountMemberBasicConfig(rnd, fmt.Sprintf("%s@domain.com", rnd)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "email_address", fmt.Sprintf("%s@domain.com", rnd)),
```

Comment:
let's use example.com here for the domain as it's available for test/documentation purposes

```suggestion
				Config: testCloudflareAccountMemberBasicConfig(rnd, fmt.Sprintf("%s@example.com", rnd)),
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr(name, "email_address", fmt.Sprintf("%s@example.com", rnd)),
```